docs/source/storage/discovery_storage.md

```diff
@@ -2,48 +2,51 @@
 
 # Data Storage Options
 
-RC is responsible for the procurement and ongoing maintenance of several data storage options, including active, and archive storage solutions. If you are affiliated with Northeastern, you can request one or more storage solutions to meet your storage needs. If you anticipate needing storage as part of a grant requirement, please [schedule a storage consultation] with an RC staff member to understand what storage options would best meet your research needs.
+RC is responsible for procuring and maintaining several data storage options, including active and archive storage solutions. If affiliated with Northeastern University, you can request one or more storage solutions to meet your needs. If you anticipate needing storage as part of a grant requirement, please [schedule a storage consultation] with an RC staff member to understand what storage options best meet your research needs.
 
 ## Active Storage
 
-There are two main storage systems connected to Northeastern's HPC cluster: `/home` and `/scratch`; these options have specific quotas and limitations. The list below details the storage options available to you on the HPC cluster by having an account. Every individual with an account has both a `/home` and `/scratch`. While research groups can request additional storage on the `/work` storage system, `/work` storage is not currently provisioned to individuals.
+**Updated 2023-07-18**
+
+Two main storage systems are connected to Northeastern's HPC cluster: `/home` and `/scratch`; these options have specific quotas and limitations. The list below details the storage options available on the HPC cluster if you have an account. Every individual with an account has a `/home` and `/scratch`. While research groups can request additional storage on the `/work` storage system, `/work` storage is not currently provisioned for individuals.
```

Comment:
Maybe change this sentence a little bit. consider this: The list below details the storage options available on the HPC cluster for users with an account.